misk-launchdarkly-core/src/main/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -75,6 +78,22 @@ class LaunchDarklyFeatureFlags @Inject constructor(
     return java.lang.Enum.valueOf(clazz, result.value.toUpperCase())
   }
 
+  override fun <T> getJson(
+    feature: Feature,
+    key: String,
+    clazz: Class<T>,
+    attributes: Attributes
+  ): T {
+    checkInitialized()
+    val result = ldClient.jsonValueVariationDetail(
+        feature.name,
+        buildUser(feature, key, attributes),
+        LDValue.ofNull())
```

Comment:
Named arg for this null arg?

Reply:
BTW, this PR isn't the place for this debate, but the lack of in-code default values for flags is still hard to get used to. As long as nobody resolved flags during app startup, and only certain code paths will fail due to unregistered flags, I guess it could be worse!